index.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+
+module.exports = class LambdaInvokeAsync {
+  constructor(options) {
+    options = options || {};
+    this._client = new AWS.Lambda();
+    this.logger = options.logger;
+  }
+
+  _formatClientContext(context) {
+    if (!context) {
+      return null;
+    }
+    return new Buffer(JSON.stringify(context)).toString('base64');
+  }
+
+  invokeAsync(functionName, payload, invocationType, clientContext) {
+    if (!functionName || !payload) {
+      throw new Error('functionName and payload are required parameters.');
+    }
+    invocationType = invocationType || 'RequestResponse'
```

Comment:
hmmm, I guess since there are already other libraries to handle other types, maybe I will lock this one down to 'Event' only. Does that seem reasonable?